bin/docker-local-install

```diff
@@ -6,26 +6,26 @@ require __DIR__ . '/../../../autoload.php';
 use Symfony\Component\Filesystem\Filesystem;
 
 const SOURCE_DIRECTORY = 'vendor/emakinafr/docker-magento2';
-const TARGET_DIRECTORY = 'docker/local';
+const DOCKER_DIRECTORY = 'docker/local';
```

Comment:
I don't see any added value to change that name. Source/Target seems pretty clear to me.

Reply:
I take the opportunity to harmonize with the other environment (docker-proximis).